cd/manager/jobs/smoke.go

```diff
@@ -9,33 +9,72 @@ import (
 	"github.com/3box/pipeline-tools/cd/manager"
 )
 
+// Allow up to 1 hours for smoke tests to run
+const SmokeTestFailureTime = 1 * time.Hour
+const SmokeTaskIdParam = "id"
+
+const ECSCluster = "ceramic-qa-tests"
```

Comment:
Might be worth moving these to `models.go` too? Just a thought.
```
const TestsCluster = "ceramic-qa-tests"
const TestsNetworkConfigurationParameter = "network_configuration"

const (
    TestType_Smoke string = "smoke"
    TestType_E2e string = "e2e"
)
```
and then the code in `smoke.go` becomes:
```
if id, err := s.d.LaunchTask(
	manager.TestsCluster,
	manager.TestsCluster+"-"+manager.TestType_Smoke+"-"+s.env,
	manager.TestsCluster+"-"+manager.TestType_Smoke,
	"/"+manager.TestsCluster+"-"+manager.TestType_Smoke+"/"+manager.TestsNetworkConfigurationParameter,
	nil); err != nil {
```
Something like that. Just a thought. Could also use `fmt.sprintf` to make the string formats a little more explicit and readable. e.g.
```
fmt.Sprintf("%s-%s-%s", manager.TestsCluster, manager.TestType_Smoke, s.env) // for family

fmt.Sprintf("/%s-%s/%s", manager.TestsCluster, manager.TestType_Smoke, manager.TestsNetworkConfigurationParameter) // for n/w config
```

Reply:
I think the benefit of having it typed out when you read the code is pretty large

As in, when I see `fmt.Sprintf("%s-%s--%s", manager.TestsCluster, manager.TestType_Smoke, s.env)` it will take me a minute, and flipping back and forth between files, to figure out what the resulting string actually is

vs if I can see `const ECSFamilyPrefix = "ceramic-qa-tests-smoke--"` and `ECSFamilyPrefix+s.env,` on the same page. It's much more obvious what the vaule is.

Reply:
pulling out `const TestsCluster = "ceramic-qa-tests"` makes sense for next pr when we migrate the e2e tests

Reply:
Ok, sg. Should take `ECS` out of the literal names though to keep the abstractions cleaner. The jobs don't care that ECS is being used under the covers.